lua/treesitter-matchup/internal.lua

```diff
@@ -155,7 +155,7 @@ function M.containing_scope(node, bufnr, key)
 end
 
 local function _node_text(node, bufnr)
-  local text = ts.query.get_node_text(node, bufnr)
+  local text = ts.get_node_text(node, bufnr)
```

Comment:
I think, simple replace will break code for versions `< 0.9+`
Maybe will be best add something like that:
```lua
local get_text = ts.get_node_text or ts.query.get_node_text
```